### PR TITLE
Pull for Mustang work, on-taskgraph merge into upstream RackHD , enables Microservices integration

### PR DESCRIPTION
**Pull request by rahmanmuhamad**
jenkins: depends on https://github.com/RackHD/on-core/pull/276, https://github.com/RackHD/on-tasks/pull/445

### CODE REVIEW

File: lib/task-runner.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ function taskRunnerFactory(
         this.subscriptions = [];
         this.running = false;
         this.activeTasks = {};
-        this.lostBeatLimit = options.lostBeatLimit || 3;
+        this.lostBeatLimit = options.lostBeatLimit || 10;
```
Comment (anhou) [inline lib/task-runner.js]: any background why change this value?

File: lib/graphs/dell-wsman-config-services-graph.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2016, Dell EMC, Inc.
```
Comment (anhou) [inline lib/graphs/dell-wsman-config-services-graph.js]: please use the latest statement '// Copyright © 2016-2017 Dell Inc. or its subsidiaries.  All Rights Reserved.' the same for other files

File: lib/graphs/dell-wsman-config-services-graph.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Dell Wsman Config Services',
+    injectableName: 'Graph.Dell.Wsman.ConfigServices',
+    options: {
+        defaults: {
+        	configServer: 'http://127.0.0.1:9600'
```
Comment (anhou) [inline lib/graphs/dell-wsman-config-services-graph.js]: indents, I see lots of indents problem in other files, please have a clean for all of them

File: lib/graphs/dell-wsman-create-repo-graph.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Dell WSMAN Create Firmware Repo Graph',
+    injectableName: 'Graph.Dell.Wsman.Create.Repo',
+    options: {
+        defaults: {
+        	catalogFilePath : null,
```
Comment (anhou) [inline lib/graphs/dell-wsman-create-repo-graph.js]: indent

File: lib/graphs/dell-wsman-discovery-graph.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+// Copyright 2016, DELL, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Dell Wsman Discovery',
+    injectableName: 'Graph.Dell.Wsman.Discovery',
+    options: {
+    	defaults: {
+    		ranges:[{
```
Comment (anhou) [inline lib/graphs/dell-wsman-discovery-graph.js]: indent

**Comment by anhou** [comment]
@rahmanmuhamad Could you help change titile to be more meaningful and also add some descritpions in the comments for what have done, it will help a lot for code review

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

**Comment by geoff-reid** [comment]
test this please

File: lib/task-runner.js (hunk 1 above)
Comment (rahmanmuhamad) [inline lib/task-runner.js]: Reverted this change

**Pull request merged**